gatling-jms/src/main/scala/io/gatling/jms/action/JmsReqReply.scala

```diff
@@ -58,7 +59,7 @@ class JmsReqReplyActor(attributes: JmsAttributes, protocol: JmsProtocol, tracker
     extends ValidatedActionActor {
 
   // Create a client to refer to
-  val client = JmsClient(protocol, attributes.destination, attributes.replyDestination)
+  val client: JmsReplyClient = JmsClient(protocol, attributes.destination, attributes.replyDestination.get).asInstanceOf[JmsReplyClient]
```

Comment:
I just marked the changed which I already done localy and wrote the comment...will push the changes soon :+1: